src/decays/specializations/Ah/decay_Ah_to_gg.inc

```diff
@@ -87,5 +95,16 @@ double CLASSNAME::get_partial_width<PseudoscalarHiggs, Gluon, Gluon>(
       );
    }
 
+   if (static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::call_higgstools)) ||
```

Comment:
See my comment on `static_cast<bool>`

Reply:
Done in c587517522b7b65208aa28eb3ac3b70a03a3ce92.